app/definition/ApiDefinitionFactory.scala

```diff
@@ -52,20 +52,22 @@ class ApiDefinitionFactory @Inject()(appConfig: AppConfig) extends Logging {
         context = appConfig.apiGatewayContext,
         categories = Seq("INCOME_TAX_MTD"),
         versions = Seq(
-          APIVersion(
-            version = VERSION_1,
-            status = buildAPIStatus(VERSION_1),
```

Comment:
does this not also need the status and endpoints enabled?

Reply:
Yes thanks - have just created sub-task to look at environment configs. (Deliberately set v1 & v2 same config in here for clarity.)